pilot/pkg/proxy/envoy/v2/monitoring.go

```diff
@@ -117,14 +117,14 @@ var (
 	// only supported dimension is millis, unfortunately. default to unitdimensionless.
 	proxiesQueueTime = monitoring.NewDistribution(
 		"pilot_proxy_queue_time",
-		"Time a proxy is in the push queue before being dequeued.",
+		"Time in milliseconds, a proxy is in the push queue before being dequeued.",
 		[]float64{.1, 1, 3, 5, 10, 20, 30},
 	)
 
 	// only supported dimension is millis, unfortunately. default to unitdimensionless.
 	proxiesConvergeDelay = monitoring.NewDistribution(
 		"pilot_proxy_convergence_time",
-		"Delay between config change and a proxy receiving all required configuration.",
+		"Delay in milliseconds between config change and a proxy receiving all required configuration.",
```

Comment:
I don't think this is true, I thought it was seconds?

Reply:
actually I'm not really sure. I think we just do `time.Since` and based on the comment above this probably is recorded as millis. Thanks for the docs

Reply:
@howardjohn Sorry, My mistake - actually you are right. It is in seconds. Verified https://github.com/istio/istio/blob/master/pilot/pkg/proxy/envoy/v2/ads.go#L620 and https://github.com/istio/istio/blob/master/pilot/pkg/proxy/envoy/v2/discovery.go#L420